addons/rose/addon/components/rose/dropdown/index.stories.mdx

```diff
@@ -148,30 +169,31 @@ form fields.
 
 ```html
 <Rose::Form as |form|>
-  <Rose::Dropdown @text="Items" @ignoreClickInside={{true}} as |dropdown|>
-<Rose::Form::Checkbox::Group
-  @name='checkbox-group'
-  @items={{this.items}}
-  @selectedItems={{this.selectedItems}}
-  @onChange={{fn this.onChange}}
-  as |group|
->
-<dropdown.item>
-  <group.checkbox
-    @label={{group.item.name}}
-    value={{group.item.id}}
-    />
-    </dropdown.item>
-</Rose::Form::Checkbox::Group>
-
- </Rose::Dropdown>
+  <Rose::Dropdown @text="Items" @ignoreClickInside="{{true}}" as |dropdown|>
+    <Rose::Form::Checkbox::Group
+      @name="checkbox-group"
+      @items="{{this.items}}"
+      @selectedItems="{{this.selectedItems}}"
+      @onChange="{{fn"
+      this.onChange}}
+      as
+      |group|
+    >
+      <dropdown.item>
+        <group.checkbox
+          @label="{{group.item.name}}"
```

Comment:
Unfortunately I think prettier doesn't know how to handle this, it thinks it's html and that the `@label` is a normal html attribute and is wrapping with it in quotes instead of leaving it alone.

I couldn't find a handlebars format but right now prettier is messing up the double curly braces so we might need to undo the formatting or figure out how to handle the handlebars format

Reply:
@ZedLi All the format was automatically applied it was nothing I did.

I need this change ASAP to keep working on storybook upgrades (this is blocking). It is possible you take care of formating issues in another PR? or what we can do to not add formating noise to this PR?

Thank you.

Reply:
I think the easiest would be to reset to the changes you had before the commit through reflog and just skip the formatting, I think we have to figure out how to handle the html or htmlbars templating as unfortunately it doesn't work well right now

Reply:
I was able to figure out how to get syntax highlighting correctly with handlebars in storybook but unfortunately can't undo your formatting changes, you'll have to undo those and I can cleanup the formatting in a different PR.

Reply:
I can fix the errors in the next formatting PR